src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -216,9 +220,33 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
+    <EmbeddedResource Include="Resources\Strings.de.resx" />
+    <EmbeddedResource Include="Resources\Strings.es.resx" />
+    <EmbeddedResource Include="Resources\Strings.fr.resx" />
+    <EmbeddedResource Include="Resources\Strings.it.resx" />
+    <EmbeddedResource Include="Resources\Strings.ja.resx" />
+    <EmbeddedResource Include="Resources\Strings.ko.resx" />
+    <EmbeddedResource Include="Resources\Strings.ru.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hans.resx" />
+    <EmbeddedResource Include="Resources\Strings.zh-Hant.resx" />
```

Comment:
I think that we do expect them all to have the same languages.  I chose these languages, which are called the VS9, because they are what @markwilkie said we should use for now.  My impression from the conversation with him is that that's subject to change.

How are you thinking I would add this list to the localization.targets file?  Would I put the EmbeddedResource and XliffResource elements directly in there, using something like "$(MSBuildProjectName).de.xlf" for the name of the XliffResources?